core/templates/dev/head/components/create_activity_modal.html

```diff
@@ -28,7 +28,7 @@ <h1 translate="I18N_CREATE_EXPLORATION_TITLE"></h1>
             <p translate="I18N_CREATE_EXPLORATION_QUESTION"></p>
           </div>
           <div class="col-sm-12 col-xs-12 text-center">
-            <img src="/images/activity/exploration.svg" ng-click="chooseExploration()">
+            <img ng-src="<[explorationImgUrl]>" ng-click="chooseExploration()">
```

Comment:
Sean suggested that we should use ng-src in directives, jinja interpolation should only be used in the 'page'-level html files and that we are phasing it out. See: https://github.com/oppia/oppia/pull/2208#discussion_r69385567